build.gradle

```diff
@@ -423,16 +442,23 @@ publishing {
             artifactId project.archivesBaseName
             version project.version
 
-            artifact stagingJar
             artifact sourceJar
             artifact javadocJar
```

Comment:
The `artifact stagingJar` and `artifact shadowJar` seems get lost in merge 05a076f5773c4b950ec0f6f2d2f0190d0e73cf97?

Reply:
I imagine so, I rushed that merge quite badly. Should be able to get this PR in quite soon.